chat-components/src/common/Constants.ts

```diff
@@ -176,6 +176,7 @@ export const Texts = class {
     public static readonly ConfirmationPaneTitle = "Close chat";
     public static readonly ConfirmationPaneSubtitle = "Do you really want to close this chat?";
     public static readonly ConfirmButtonText = "Close";
+    public static readonly EmailPlaceHolderText = "johnsmith@outlook.com";
```

Comment:
Is this used somewhere else ? Just curious if it is someone's personal email address, it can cause security/privacy concerns!!

Reply:
Is not , it was provided by PM